lib/stsci/tools/nmpfit.py

```diff
@@ -1,5 +1,5 @@
 """
-Python/Numeric version of this module was called mpfit. This version was modified to use numpy.
+Python/Numeric version of this module was called mpfit. This version was modified to use np.
```

Comment:
Your `sed` got a little too aggressive? 
```suggestion
Python/Numeric version of this module was called mpfit. This version was modified to use numpy.
```